homeassistant/components/air_quality/nilu.py

```diff
@@ -0,0 +1,268 @@
+
+    @property
+    def attribution(self) -> str:
+        """Return the attribution."""
+        return CONF_ATTRIBUTION
+
+    @property
+    def device_state_attributes(self) -> dict:
+        """Return other details about the sensor state."""
+        return self._attrs
+
+    @property
+    def icon(self) -> str:
+        """Icon to use in the frontend."""
+        return ICON
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def air_quality_index(self) -> str:
+        """Return the Air Quality Index (AQI)."""
+        return self._max_aqi
+
+    @property
+    def carbon_monoxide(self) -> str:
+        """Return the CO (carbon monoxide) level."""
+        from niluclient import CO
+        return self.get_component_state(CO)
+
+    @property
+    def carbon_dioxide(self) -> str:
+        """Return the CO2 (carbon dioxide) level."""
+        from niluclient import CO2
+        return self.get_component_state(CO2)
+
+    @property
+    def nitrogen_oxide(self) -> str:
+        """Return the N2O (nitrogen oxide) level."""
+        from niluclient import NOX
+        return self.get_component_state(NOX)
+
+    @property
+    def nitrogen_monoxide(self) -> str:
+        """Return the NO (nitrogen monoxide) level."""
+        from niluclient import NO
+        return self.get_component_state(NO)
+
+    @property
+    def nitrogen_dioxide(self) -> str:
+        """Return the NO2 (nitrogen dioxide) level."""
+        from niluclient import NO2
+        return self.get_component_state(NO2)
+
+    @property
+    def ozone(self) -> str:
+        """Return the O3 (ozone) level."""
+        from niluclient import OZONE
+        return self.get_component_state(OZONE)
+
+    @property
+    def particulate_matter_2_5(self) -> str:
+        """Return the particulate matter 2.5 level."""
+        from niluclient import PM25
+        return self.get_component_state(PM25)
+
+    @property
+    def particulate_matter_10(self) -> str:
+        """Return the particulate matter 10 level."""
+        from niluclient import PM10
+        return self.get_component_state(PM10)
+
+    @property
+    def particulate_matter_0_1(self) -> str:
+        """Return the particulate matter 0.1 level."""
+        from niluclient import PM1
+        return self.get_component_state(PM1)
+
+    @property
+    def sulphur_dioxide(self) -> str:
+        """Return the SO2 (sulphur dioxide) level."""
+        from niluclient import SO2
+        return self.get_component_state(SO2)
+
+    def get_component_state(self, component_name: str) -> str:
+        """Return formatted value of specified component."""
+        if component_name in self._api.data.sensors:
+            sensor = self._api.data.sensors[component_name]
+            return "{0:.2f} {1}".format(sensor.value,
+                                        sensor.unit_of_measurement)
```

Comment:
There should preferably not be a unit of measurement but that should be standardized. Maybe we haven't done that yet? @fabaff

Reply:
Is it possible to add en property where it gives the unit of measurement, such as on sensors? I was not sure if all measured components should be defined in `µg/m³`, so I added it to the output of each.  

All measured pollutants currently supported from the NILU-client is measured in `µg/m³` or `mg/m³`, so one fit it all is possible with what the `air_quality` component gives of information now. `CO`/carbon monoxide is the one thats measured in `mg/m³`, but this can easily be converted to `µg/m³` to get a one fits it all.

Can add this to the platform:
```python
    @property
    def unit_of_measurement(self) -> str:
        """Return the unit this state is expressed in."""
        return 'µg/m³'
```

Reply:
There should be no unit of measurements because the frontend should take care about this.

Reply:
I will fix the output. What is the default unit the frontend expect, so I can make sure they are correct or converted?